frame/multisig/src/lib.rs

```diff
@@ -257,9 +257,9 @@ pub mod pallet {
 			let dispatch_info = call.get_dispatch_info();
 			(
 				T::WeightInfo::as_multi_threshold_1(call.using_encoded(|c| c.len() as u32))
-					.saturating_add(dispatch_info.weight)
 					// AccountData for inner call origin accountdata.
-					.saturating_add(T::DbWeight::get().reads_writes(1, 1)),
+					.saturating_add(T::DbWeight::get().reads_writes(1, 1))
+					.saturating_add(dispatch_info.weight),
```

Comment:
Does the ordering here really matter?

Reply:
no, this is left over from some other changes where it did matter.

Reply:
should be superficial, so not gonna go back to fix